scripts/generate-accounts/README.md

```diff
@@ -0,0 +1,38 @@
+# Add Validator
+
+This script will help us in generating accounts programmaticaly. 
+
+<br />
+
+## Requirements
+[NodeJs (v14.17.0 lts)](https://nodejs.org/en/download/ "NodeJs (v14.17.0 lts)")
```

Comment:
Why?

Reply:
We can use when we run it in local machine without using docker

Reply:
Take a [look](https://github.com/Cerebellum-Network/nodes-installation-scripts/pull/49#discussion_r655489234).

Reply:
Updated with docker version